app/_src/quickstart/kubernetes-demo.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Deploy Kuma on Kubernetes
+---
+
+To start learning how {{site.mesh_product_name}} works, you run and secure a simple demo application that consists of two services:
+
+- `demo-app`: a web application that lets you increment a numeric counter. It listens on port 5000
+- `redis`: data store for the counter
+
+## Prerequisites
+- [Helm](https://helm.sh/) - a package manager for Kubernetes
+- [Kind](https://kind.sigs.k8s.io/) - a tool for running local Kubernetes clusters
+
+## Start Kubernetes cluster
+
+Start a new Kubernetes cluster on your local machine by executing:
+
+```sh
+kind create cluster --name=kuma
+```
+
+{% tip %}
+You can skip this step if you already have a Kubernetes cluster running.
+It can be a cluster running locally or in a public cloud like AWS EKS, GCP GKE, etc.
+{% endtip %}
+
+## Install Kuma
+
+Install Kuma control plane with Helm by executing:
+
+```sh
+helm repo add kuma https://kumahq.github.io/charts
+helm repo update
+helm install --create-namespace --namespace {{site.mesh_namespace}} {{ site.kuma_install_name }} {{ site.mesh_helm_repo }}
```

Comment:
Maybe just me but when I follow the instructions when I execute this one I get:

```bash
Error: INSTALLATION FAILED: must either provide a name or specify --generate-name
```

My `helm` is `Version:"v3.8.2"`

Reply:
Stupid question. Did you execute exactly this above or after interpolation
```
helm install --create-namespace --namespace kuma-system  kuma/kuma
```
?

Reply:
No prob and thanks for checking!

I executed exactly:

```bash
helm install --create-namespace --namespace kuma-system  kuma/kuma
```

I just did it again to be double sure 😅 

I guess its a "just me" thing, I wondered if it was down to my older helm version?


![Screenshot 2024-01-23 at 14 55 40](https://github.com/kumahq/kuma-website/assets/554604/b365cefa-9f70-4ded-b972-caa1c41a12e5)

Reply:
Resolving this one